general-terms-of-use.md

```diff
@@ -1,48 +1,48 @@
 # Clever General Terms of Use
 
-_Effective Date: May 4, 2016_
+_Effective Date: Jan 6, 2017_
 
 Welcome to Clever! We’re delighted to have you on our website and maybe even using our service (the “Services”). Please continue reading to learn about the terms by which you may use our Services. If you have questions about this Agreement, please contact us at [support@clever.com](mailto:support@clever.com).
 
-These General Terms of Use (the “Agreement”) applies to all schools, school districts, and related entities and organizations that sign up to use the Services, including but not limited to administrators who access the Services on their behalf (each a “School”), as well as all non-School visitors, users, and others, including students, teachers, and their parents, who use the Services (such individuals and Schools, collectively, “Users” or “you”). If you’re a School, our [Additional Terms of Use for Schools](https://clever.com/about/terms-schools) apply to you too.
+These General Terms of Use apply to all: (i) schools, school districts, and related entities and organizations that sign up to use the Services, including but not limited to administrators who access the Services on their behalf (each a “School”); (ii) non-School visitors, users, and others, including students, teachers, and their parents, who use the Services; and (iii) third-party providers of applications (each a “Developer” and collectively with  the individuals referenced in (ii) above and Schools, the “Users” or “you”). If you’re a School, our [Additional Terms of Use for Schools](https://clever.com/about/terms-schools) apply to you too. If you’re a Developer, our [Additional Terms of Use for Developers](https://clever.com/about/terms-developers) apply to you too. These General Term of Use, the [Additional Terms of Use for Schools](https://clever.com/about/terms-schools) (as applicable to Schools) and the [Additional Terms of Use for Developers](https://clever.com/about/terms-developers) (as applicable to Developers) are collectively referred to herein as the “Agreement”.
```

Comment:
Last sentence in this paragraph should say "These General Terms of Use, ..." instead of "These General Term of Use, ..." (add an s to Term)